.github/workflows/publish.yml

```diff
@@ -40,8 +40,8 @@ jobs:
           always-auth: true
           registry-url: "https://registry.npmjs.org"
 
-      # Perform installs, run tests, run a build step, etc. here, as needed.
-      - run: yarn
+      # Perform installs and run a build step.
+      - run: yarn install --frozen-lockfile && yarn build
```

Comment:
With removed `prepublish` script we have to build explicitly. For now this includes regular SVM build without caching, but would need to create a follow-up PR that covers verified builds.